src/NHibernate/Criterion/Lambda/QueryOverProjectionBuilder.cs

```diff
@@ -50,6 +48,15 @@ public QueryOverProjectionBuilder<T> WithAlias(Expression<Func<object>> alias)
 			lastProjection = Projections.Alias(lastProjection, aliasContainer);
 			return this;
 		}
+		
+		/// <summary>
+		/// Create an alias for the previous projection
+		/// </summary>
+		public QueryOverProjectionBuilder<T> WithAlias(string alias)
```

Comment:
This small addition is not related to fix. Just added it for parity with `ProjectionsExtensions.WithAlias` methods